pages/workflow/PKPWorkflowHandler.php

```diff
@@ -625,6 +625,11 @@ public function editorDecisionActions($args, $request)
                     }
                     $allRecommendations = join(__('common.commaListSeparator'), $allRecommendations);
                 }
+            } elseif ($stageId == WORKFLOW_STAGE_ID_SUBMISSION) {
+                    /** @var StageAssignmentDAO $stageAssignmentDao */
+                    $stageAssignmentDao = DAORegistry::getDAO('StageAssignmentDAO');
+                    $decidingEditorIds = $stageAssignmentDao->getDecidingEditorIds($submission->getId(), $stageId);
+                    $hasDecidingEditors = count($decidingEditorIds) > 0;
```

Comment:
The behaviour looks different than in 3.3. Should we allow sending to the review stage even if deciding editors aren't assigned?